jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/model/ProcessAssetDesc.java

```diff
@@ -210,6 +215,23 @@ public void setSignals(Collection<String> signals) {
         this.signals = signals;
     }
 
+    @Override
+    public Collection<SignalDesc> getSignalsMetadata() {
```

Comment:
same as before. change to a more descriptive name. this is not the metadata but  the element itself.